upi/openstack/common.yaml

```diff
@@ -2,25 +2,32 @@
   gather_facts: no
 
   vars_files:
-  - metadata.json
+  - infraid.json
 
   tasks:
   - name: 'Compute resource names'
     set_fact:
-      cluster_id_tag: "openshiftClusterID={{ infraID }}"
-      primary_cluster_network_tag: "{{ infraID }}-primaryClusterNetwork"
-      os_infra_id: "{{ infraID }}"
-      os_port_bootstrap: "{{ infraID }}-bootstrap-port"
-      os_port_master: "{{ infraID }}-master-port"
-      os_port_worker: "{{ infraID }}-worker-port"
+      cluster_id_tag: "openshiftClusterID={{ infra_id }}"
```

Comment:
This would mean we stopped tagging the resources with the ID generated by openshift, instead we would be tagging some an ID that was generated on the step ` Deployment identifier`.
I believe it's better to keep using the openshift identified for the `cluster_id_tag`, reason being that this is what is available in the `metadata.json` and if the customer wants to do a `openshift-install cluster destroy` that file will be used, also there might be other component that rely on that identifier.

Reply:
For the UPI installation it doesn't matter if OpenStack resources are tagged with OpenShift cluster id or anything else. In UPI we don't use `openshift-install cluster destroy` but the playbooks, and playbooks remove OpenStack resources based only on that specific tag. Could you specify what components in OpenShift depends this tag in context of OpenStack resources?

Reply:
I've seen QE using the destroy command once with UPI, not sure if they still do, maybe @eurijon does?

Reply:
That's a good point. I'll live all the tagging like it was before, but for network/subnet(s) and router there will be a need to keep `netid.json` file created during installation.

Reply:
Hey, we use the `down-*` playbooks for destroying UPI clusters